diff --git a/core/browser.go b/core/browser.go
index 5c6363c..8ae5f08 100644
--- a/core/browser.go
+++ b/core/browser.go
@@ -225,7 +225,9 @@ func NewBrowser(opts BrowserOpts) (*Browser, error) {
 		Set("disable-blink-features", "AutomationControlled").
 		Delete("enable-automation").
 		Set("use-angle", "swiftshader-webgl").
-		Set("ignore-gpu-blocklist")
+		Set("ignore-gpu-blocklist").
+		Set("no-sandbox").
+		Set("disable-setuid-sandbox")
 	if opts.IsHeadless {
 		l = l.HeadlessNew(true)
 	} else {
